Gateway/Request/APMBuilder.php

```diff
@@ -82,6 +88,12 @@ public function build(array $buildSubject)
                     self::SOURCE_INSTALLMENT_TERMS => $method->getAdditionalInformation(InstallmentDataAssignObserver::TERMS),
                 ];
                 break;
+            case Truemoney::CODE:
+                $paymentInfo[self::SOURCE] = [
+                    self::SOURCE_TYPE      => 'truemoney',
+                    self::SOURCE_PHONE_NUM => $method->getOrder()->getBillingAddress()->getTelephone(),
```

Comment:
I'm not sure if using phone number from billing address would be practical for this True Money case.
Should we have an input to let user fill their phone number by themselves? Or at least having a choice to use a phone number from the billing address or enter a new one.

Reply:
that might be a good idea, I will consider to add it in the future PR.

Reply:
👍 okie